docs/config/README.md

```diff
@@ -1726,16 +1726,17 @@ The `git_commit` module shows the current commit hash and also the tag (if any)
 
 ### Options
 
-| Option               | Default                      | Description                                                                          |
-| -------------------- | ---------------------------- | ------------------------------------------------------------------------------------ |
-| `commit_hash_length` | `7`                          | The length of the displayed git commit hash.                                         |
-| `format`             | `'[\($hash$tag\)]($style) '` | The format for the module.                                                           |
-| `style`              | `'bold green'`               | The style for the module.                                                            |
-| `only_detached`      | `true`                       | Only show git commit hash when in detached `HEAD` state                              |
-| `tag_disabled`       | `true`                       | Disables showing tag info in `git_commit` module.                                    |
-| `tag_max_candidates` | `0`                          | How many commits to consider for tag display. The default only allows exact matches. |
-| `tag_symbol`         | `' 🏷 '`                      | Tag symbol prefixing the info shown                                                  |
-| `disabled`           | `false`                      | Disables the `git_commit` module.                                                    |
+| Option                     | Default                      | Description                                                                          |
+| -------------------------- | ---------------------------- | ------------------------------------------------------------------------------------ |
+| `commit_hash_length`       | `7`                          | The length of the displayed git commit hash.                                         |
+| `format`                   | `'[\($hash$tag\)]($style) '` | The format for the module.                                                           |
+| `style`                    | `'bold green'`               | The style for the module.                                                            |
+| `only_detached`            | `true`                       | Only show git commit hash when in detached `HEAD` state                              |
+| `tag_disabled`             | `true`                       | Disables showing tag info in `git_commit` module.                                    |
+| `tag_disabled_on_detached` | `false`                      | Disables showing tag info in detached `HEAD` state.                                  |
```

Comment:
Isn't this PR supposed to allow showing the tag with `only_detached`-enabled in an attached/non-detached state?